modules/styling/lib/cs.ts

```diff
@@ -1211,6 +1239,11 @@ export function createStencil<
     };
   }) as any;
 
+  stencil.parts = (composes?.__parts ? {...composes.__parts, ..._parts} : _parts) as [E] extends [
```

Comment:
This doesn't look correct. The `__parts` is a type and not a value. For most of the other generics, the type is a fundamental type (the type isn't directly used as a value, but a value is derived from the type) and not representative of any value. I thought there was a JSDoc warning here, but I don't see it. The `__id` could be a value since it is always a fundamental type, but we don't expose it. The `__extends` is both a fundamental type and a value we add to the Stencil although I don't see it being used anywhere. It could be used to do lookups of where `parts`, `vars`, or any other thing comes from. We tend to merge everything instead of lookups later.